src/Formats/ProtobufSerializer.cpp

```diff
@@ -1722,7 +1726,7 @@ namespace
         String text_buffer;
     };
 
-    using ProtobufSerializerIPv4 = ProtobufSerializerNumber<UInt32>;
+    using ProtobufSerializerIPv4 = ProtobufSerializerNumber<IPv4>;
```

Comment:
Shouldn't this serializer be more elaborated? (see `ProtobufSerializerIPv6`)

Reply:
I also thought about it. Previously `IPv4` type was jsut a custom type for `UInt32`. And only recently we made it a separate type. I just thought that for compatibility reasons we should read/write it as it was before just like `UInt32`. 
But maybe it's not really the case. I will make a separate serializer for it (it will be really better)

Reply:
Done